ReactiveCocoa/UIKit/UIControl.swift

```diff
@@ -44,11 +44,23 @@ extension Reactive where Base: UIControl {
 	/// - parameters:
 	///   - controlEvents: The control event mask.
 	///
-	/// - returns: A signal that sends the control each time the control event 
-	///            occurs.
+	/// - returns: A signal that sends the control each time the control event occurs.
 	public func controlEvents(_ controlEvents: UIControlEvents) -> Signal<Base, NoError> {
```

Comment:
There are control events that are not value affecting though, e.g. `editingDidBegin`.